docs/_includes/component/footer.njk

```diff
@@ -13,21 +13,30 @@
     </div>
   </section> #}
 
-  <footer class="main-footer">
-    <img style="max-width: 135px;" src="https://static.redhat.com/libs/redhat/brand-assets/2/corp/logo--on-dark.svg" alt="Red Hat" />
-    <p>Copyright &copy; 2021-{{ page.date.getFullYear() }} Red Hat, Inc.</p>
-    <div class="main-footer--list">
-      <ul>
-        <li><a href="https://www.redhat.com/en/about/privacy-policy">Privacy statement</a></li>
-        <li><a href="https://www.redhat.com/en/about/terms-use">Terms of use</a></li>
-        <li><a href="https://www.redhat.com/en/about/all-policies-guidelines">All policies and guidelines</a></li>
-      </ul>
-    </div>
-
-    <a id="netlify-badge" href="https://www.netlify.com" target="_blank" rel="noreferrer noopener">
-      <img src="https://www.netlify.com/img/global/badges/netlify-color-accent.svg" alt="Deploys by Netlify">
```

Comment:
We must include the netlify link

Reply:
Added the netlify link to the tertiary region of the universal footer.